docs/zh/latest/how-to-build.md

```diff
@@ -36,22 +36,22 @@ Apache APISIX 的运行环境需要 Nginx 和 etcd，
 你需要先下载 Apache Release 源码包：
 
 ```shell
-$ mkdir apisix-2.3
-$ wget https://downloads.apache.org/apisix/2.3/apache-apisix-2.3-src.tgz
-$ tar zxvf apache-apisix-2.3-src.tgz -C apisix-2.3
+$ mkdir apisix-2.4
+$ wget https://downloads.apache.org/apisix/2.4/apache-apisix-2.4-src.tgz
+$ tar zxvf apache-apisix-2.4-src.tgz -C apisix-2.4
 ```
 
 安装运行时依赖的 Lua 库：
 
 ```
-cd apache-apisix-2.3
+cd apache-apisix-2.4
```

Comment:
`apache-apisix-2.4` => `apisix-2.4`

Reply:
It's better to fix it in master branch and cherry-pick to release branch I think